tests/spec/spec/kubernetes_master/kubernetes_master_spec.rb

```diff
@@ -207,18 +206,32 @@
   end
 end
 
-describe 'Check the docker cgroup and logging driver' do
-  describe file('/etc/docker/daemon.json') do
+describe 'Check the containerd' do
+  describe command('crictl version') do
     let(:disable_sudo) { false }
-    its(:content_as_json) { should include('exec-opts' => include('native.cgroupdriver=systemd')) }
-    its(:content_as_json) { should include('log-driver' => 'json-file') }
-    its(:content_as_json) { should_not include('exec-opts' => include('native.cgroupdriver=cgroupfs')) }
+    its(:stdout) { should include('RuntimeName:  containerd') }
+  end
+  describe command("kubectl get nodes -o jsonpath='{.items[].status.nodeInfo.containerRuntimeVersion}'") do
+    its(:stdout) { should include('containerd://1.4.12') }
```

Comment:
This test fails in case of single machine installation:
```
2022-02-14T08:39:45.1586889Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -   1) Check the containerd Command "kubectl get nodes -o jsonpath='{.items[].status.nodeInfo.containerRuntimeVersion}'" stdout is expected to include "containerd://1.4.12"
2022-02-14T08:39:45.1594302Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -      On host `1.1.1.1'
2022-02-14T08:39:45.1603596Z[31;21m08:39:45 ERROR cli.src.spec.SpecCommand -      Failure/Error: its(:stdout) { should include('containerd://1.4.12') }
2022-02-14T08:39:45.1612061Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -        expected "docker://20.10.8" to include "containerd://1.4.12"
2022-02-14T08:39:45.1617308Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -        /bin/bash -c kubectl\ get\ nodes\ -o\ jsonpath\=\'\{.items\[\].status.nodeInfo.containerRuntimeVersion\}\'
2022-02-14T08:39:45.1622739Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -        docker://20.10.8
2022-02-14T08:39:45.1630605Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -      # ./spec/kubernetes_master/kubernetes_master_spec.rb:215:in `block (3 levels) in <top (required)>'
2022-02-14T08:39:45.1633799Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand -[0m
2022-02-14T08:39:45.1640621Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand - Finished in 59.92 seconds (files took 0.46699 seconds to load)
2022-02-14T08:39:45.1648014Z[38;21m08:39:45 INFO cli.src.spec.SpecCommand - 66 examples, 1 failure
```